test/tax-lot/tax-lot.repository.mock.ts

```diff
@@ -0,0 +1,12 @@
+import { generateMock } from "@anatine/zod-mock";
+import { findByBblSpatialSchema } from "src/tax-lot/tax-lot.repository.schema";
+
+export class TaxLotRepositoryMock {
+  findByBblSpatialMocks = Array.from(Array(10), () =>
+    generateMock(findByBblSpatialSchema, { seed: 1 }),
```

Comment:
- Generate fake data from the endpoint schema.
- Place it in a publicly available variable, so that tests can access it for data calls and expected values. 
- Use a seed to make the test data deterministic